Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if (true === $mark->isRunning() && false === $mark->isPause()) {
```

Comment:
I disagree with you.
`isRunning()` is related to `->_isRunning` and `isPause()` to `->_isPause`. 
This condition should be in a other method. 

I purpose 2 choices:

1- 
  - rename /isRunning/ to `isInitialized`
  - add `isRunning()` for initilized unpaused mark.

2- 
  - add `isTiming()`

The second point is lighter ;)